Mysql/insert_test_forms.sql

```diff
@@ -0,0 +1,5 @@
+-- Dumping data for table ukraine.forms: ~1 rows (approximately)
+/*!40000 ALTER TABLE `forms` DISABLE KEYS */;
+INSERT INTO `forms` (`id`, `title`, `xform_id`) VALUES
+    ('agRjX9kiP7TSU45B8aLsUN', 'Iodine Survey - Building Listing', 'ukraine_iodine_building_listing');
```

Comment:
In my kobotoolbox the id for this form is different, probably I have an old version. In any case are you sure is correct?

Reply:
This is the id for the one attached to the ukraine_iodine account in kobo humanitarian. It'll be different if you've uploaded a version into your account. 

But I should update this script with the main survey form and pilot test forms soon...